backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -217,8 +230,11 @@ public async Task<AvailabilitySlotDto> GetAvailabilitySlotById(long id)
 
                 _context.Update(availabilitySlot);
             }
-            
-            _mapper.Map(updateAvailabilityDto.Schedule, availabilitySlot.Schedule);
+
+            if (!updateAvailabilityDto.Schedule.WithTeamMembers)
+            {
+                _mapper.Map(updateAvailabilityDto.Schedule, availabilitySlot.SlotMembers.First().Schedule);
```

Comment:
Why we simply take schedule of the first slotmember?

Reply:
If schedule isn't set to have teamMembers(which is hardcoded now) we only have one slot member(author)